test/integration-test.sh

```diff
@@ -2,13 +2,22 @@
 
 set -e
 
-# run the domain decomp tool
-${MPIEXEC} --oversubscribe ${MPIEXEC_NUMPROC_FLAG} 3 ${MPIEXEC_PREFLAGS} ../decomp -g test_1.nc -x x -y y -m mask -o 'yx'
+# run the domain decomp tool on test_1.nc
+${MPIEXEC} --oversubscribe ${MPIEXEC_NUMPROC_FLAG} 3 ${MPIEXEC_PREFLAGS} ../decomp -g test_1.nc -x x -y y -m mask -o 'yx' > tmp.log
 
 # check the two output files are identical to the reference files
 for filename in partition_mask_3 partition_metadata_3
 do
 ncdump "${filename}.nc" > "${filename}.cdl"
-diff "${filename}.cdl" "${CMAKE_CURRENT_SOURCE_DIR}/ref_${filename}.cdl"
+diff "${filename}.cdl" "${CMAKE_CURRENT_SOURCE_DIR}/test_1/ref_${filename}.cdl"
 done
 
+# run the domain decomp tool on test_2.nc
+${MPIEXEC} --oversubscribe ${MPIEXEC_NUMPROC_FLAG} 3 ${MPIEXEC_PREFLAGS} ../decomp -g test_2.nc -x m -y n -m land_mask -o 'yx' > tmp.log
```

Comment:
Are these `tmp.log`s for debugging or do you intend to keep them? If they are to be kept then perhaps rename and append with `>>` the second time.

Reply:
Sort of neither :grimacing: .... I did it to remove the output of running `decomp` from printing to screen. However, now that you have raised it, it would be better for me to redirect to `/dev/null`. I will submit a new commit doing just that.

Reply:
my reason for re-directing is it will make the error between two files easier to spot (coming from the `diff` command)

Reply:
Ah okay, got it!

Reply:
Yeah happy to merge once tests pass :+1:.